versioning/revisions.rst

```diff
@@ -76,6 +76,24 @@ Take into account that it changes the default Conan behavior. e.g:
     - If you generate and upload N binary packages for a recipe with a given revision, then if you modify the recipe, and thus the recipe
       revision, you need to build and upload N new binaries matching that new recipe revision.
 
+GIT and Line Endings on Windows
+-------------------------------
+
+.. warning::
+
+  **Problem**
+
+  Git will (by default) checkout files in Windows systems using CRLF line endings, effectively producing different files. As files are different, the Conan revisions will be different from the revisions computed in other platforms such as Linux, resulting in missing the respective binaries in the other revision. 
+
+**Solution**
+
+It is necessary to instruct Git to do the checkout with the same line endings. This can be done several ways, for example, by adding a .gitattributes file:
+
+.. code-block:: ini
+
+  [auto]
+    crlf = false
```

Comment:
I'd never recommend changing it that way (for all files). why?
- some files (e.g. `.rc` resource files) should always have CRLF
- some files (e.g. `Makefile` makefiles) should always have LF
applying global `clrf=false` or `clrf=true` will cause breakage, of one sort or another (and errors aren't very descriptive and easy to debug, e.g. `common.mak:139: *** missing separator.  Stop.`)
I'd reference another good example of `.gitattributes` file here, e.g. [this one](https://gist.github.com/dpalomar/ebc64d6f56cfc25a3e5c20fe7cdfbd4f).

Reply:
Yeah, I thought it was a simple thing. It's much more complicated than I realized.  Not sure embedding or linking to that gist is a good idea. Can you propose a cut down subset which you think would be a good suggestion?

Reply:
this was just a first example of .gitattributes I've found in google :)
I guess we may safely put minimal example, doing changes only for `*.py` (or even just for `conanfile.py` for minimal breakage)

Reply:
this should be enough IMO:
```
conanfile.py            text eol=lf
```
(maybe same for exported sources, if any, like `CMakeLists.txt`)

Reply:
wait, the original suggestion should work:
```
  [auto]
    crlf = false
```
it means just keep line endings unchanged.